Zapper-fi/studio
# feat(mean-finance): add Mean Finance Optimism Airdrop data

## Description

We added an $OP airdrop that came from a grant proposal we did some time ago. This adds information for the users that apply and haven't claimed their airdrop yet to be able to check it out on zapper :)

## Checklist

- [X] (optional) As a contributor, my Ethereum address/ENS is: fiboape.eth
- [X] (optional) As a contributor, my Twitter handle is: @fiboape

## How to test?

You can use the address `0x7AfB052ae7B80aAc6e559281cC261089738b66F6` to check that it has airdrop available.
